cwl/src/main/scala/wdl4s/cwl/AddEmbeddedCwl.scala

```diff
@@ -0,0 +1,50 @@
+package wdl4s.cwl
+
+import shapeless.{Path => _, _}
+import ammonite.ops._
+import cats.syntax.traverse._
+import cats.syntax.either._
+import cats.instances.list._
+import cats.effect.IO
+import CwlDecoder.{Parse, ParseValidated}
+import cats.{Applicative, Monad}
+import cats.data.EitherT._
+import cats.data.EitherT
+import lenthall.validation.ErrorOr._
+
+object AddEmbeddedCwl extends Poly1 {
+  val workflowStepLens = lens[Workflow].steps
+  val workflowStepRunLens = lens[WorkflowStep].run
+
+  implicit def workflow =
+    at[Workflow] {
+      wf =>
+        EitherT {
+          wf.steps.toList.
+            flatMap(_.run.select[String].toList).
+            traverse[ParseValidated, (String, Cwl)] {
+              path =>
+                CwlDecoder.
+                  decodeAllCwl(Path(path.drop(5))).
```

Comment:
Yeah... that'd probably be better as it's consistent w/ other parts of the codebase and also doesn't require this `drop` shenanigan